### PR TITLE
Hooks

### PR DESCRIPTION
**Pull request by Zokormazo**
Add support for ansistrano style hooks before and after steps

### CODE REVIEW

**Comment by Zokormazo** [comment]
PR updated to current head && using import_tasks instead of include_tasks

File: config/after_symlink_shared.yml [hunk 1]
```diff
@@ -1,17 +1,41 @@
+- import_tasks: "{{ ansistrano_symfony_before_composer_file | default('steps/empty.yml') }}"
```
Comment (scuben) [inline config/after_symlink_shared.yml]: Here it should use `include_tasks` to be able to use runtime variables in any ansistrano_symfony** variable.

**Comment by Zokormazo** [comment]
PR updated

File: config/after_symlink_shared.yml [hunk 2]
```diff
@@ -1,17 +1,41 @@
-- import_tasks: "../../cbrunnkvist.ansistrano-symfony-deploy/config/steps/composer.yml"
+- include_tasks: "{{ ansistrano_symfony_before_composer_tasks_file | default('steps/empty.yml') }}"
+
+- include_tasks: "../../cbrunnkvist.ansistrano-symfony-deploy/config/steps/composer.yml"
```
Comment (scuben) [inline config/after_symlink_shared.yml]: Almost :) Please revert the changes to the local `/steps/*.yml` files to `import_tasks` since those are static imports.
